### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by Sonya092**

### CODE REVIEW

File: src/components/Contact/ContactForm/style.jsx [hunk 1]
```diff
@@ -5,7 +5,8 @@ const SFormContact = styled.form`
   flex-direction: column;
   padding: 5px;
   width: 30vw;
-  border: 2px solid #0081c9;
+  border: 2px solid var(--mColor);
+
```
Comment (KeerhemFR) [inline src/components/Contact/ContactForm/style.jsx:9]: Will have to delete this white space

File: src/components/Contact/ContactForm/style.jsx [hunk 2]
```diff
@@ -16,22 +17,23 @@ const SFormContact = styled.form`
       margin: 5px;
       height: 2vw;
       border-radius: 5px;
-      border: 2px solid #0081c9;
+      border: 2px solid var(--mColor);
+
```
Comment (KeerhemFR) [inline src/components/Contact/ContactForm/style.jsx:21]: same

**Review by KeerhemFR — APPROVED** [review]
Don't see any evil here

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**